worlds/sc2/MissionOrders.py

```diff
@@ -0,0 +1,389 @@
+            for new_chain in range(chains_to_make):
+                campaign.chain_names.append(chain_name_options.pop())
+                campaign.last_mission_in_chain.append(campaign.last_mission_in_chain[0])
+        # Updating branches
+        for side_chain in range(len(campaign.chain_names) - 1, 0, -1):
+            campaign.add_mission(side_chain, mission_difficulty)
+        # Adding main path mission
+        current_required_missions = (len(campaign.mission_order) * 3) // 4
+        campaign.add_mission(0, mission_difficulty, current_required_missions)
+    campaign.add_mission(0, MissionPools.FINAL, current_required_missions)
+    return {SC2Campaign.GLOBAL: campaign.mission_order}
+
+
+def make_gauntlet(num_missions: int) -> List[FillMission]:
+    mission_order: List[FillMission] = []
+    row_length = 7
+    rows = math.ceil(num_missions / row_length)
+    difficulty_rate = (max_difficulty + 1) / num_missions if num_missions < 21 else 1/3
+    if rows == 1:
+        column_names = ["I", "II", "III", "IV", "V", "VI", "VII"][:num_missions - 1]
+        column_names.append("Final")
+    else:
+        column_names = [f'_{col + 1}' for col in range(row_length)]
+    first_mission = FillMission(MissionPools.STARTER, [MissionConnection(-1)], column_names[0],
+                                completion_critical=True)
+    mission_order.append(first_mission)
+    mission_number = 1
+    space_rows = 0
+    while mission_number < num_missions:
+        if mission_number == num_missions - 1:
+            difficulty = MissionPools.FINAL
+        else:
+            difficulty_progress = mission_number * difficulty_rate
+            difficulty = smooth_difficulty[min(math.floor(difficulty_progress), max_difficulty)]
+        connection = MissionConnection(mission_number - 1)
+        mission_order.append(FillMission(
+            difficulty,
+            [connection],
+            column_names[mission_number % row_length],
+            completion_critical=True,
+            ui_vertical_padding=space_rows
+        ))
+        mission_number += 1
+        # Next row
+        if mission_number == row_length:
+            space_rows += 1
+    return {SC2Campaign.GLOBAL: mission_order}
+
+
+def make_blitz(num_missions: int) -> List[FillMission]:
+    min_width, max_width = 2, 5
+    mission_divisor = 5
+    dynamic_width = num_missions / mission_divisor
+    width = math.floor(max(min(dynamic_width, max_width), min_width))
+    middle_column = math.floor(width / 2)
+    connections = [MissionConnection(-1)]
+    mission_number = 0
+    mission_order: List[FillMission] = []
+    if num_missions % width > middle_column:
+        final_row = math.floor(num_missions / width) * width
+        final_mission_number = final_row + middle_column
+    else:
+        final_mission_number = num_missions - 1
+    while mission_number < num_missions:
+        column = mission_number % width
+        row = math.floor(mission_number / width)
```

Comment:
(minor, optional) These division-and-floor operations (here and on line 291) can just be integer division like you do in `make_golden_path()`, line 244.

Reply:
Fixed.  Forgot about those.

Reply:
I think you fixed every one except the one this comment is on.

Reply:
Whoops!  Got it.